demisto_sdk/commands/split/jsonsplitter.py

```diff
@@ -124,4 +146,69 @@ def create_module(self):
 
             module_file_path = os.path.join(self.module_dir, file_name)
 
-        write_dict(module_file_path, data=self.module_json_data, indent=4)
+        write_dict(module_file_path, data=self.json_data, indent=DEFAULT_JSON_INDENT)
+
+    def get_auto_output_path(self) -> Tuple[Path, str, str]:
```

Comment:
We can simplify it and return just one object (the file path), since the folder and the data path can be calculated from it, **and** this is only used once.